src/Workspaces/CoreTest/WorkspaceTests/MSBuildWorkspaceTests.cs

```diff
@@ -315,6 +315,49 @@ public void TestOpenProject_CSharp_WithoutAssemblyName()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.Workspace)]
+        public void TestOpenProject_CSharp_WithoutCSharpTargetsImported_Succeeds()
+        {
+            CreateFiles(GetSimpleCSharpSolutionFiles()
+                .WithFile(@"CSharpProject\CSharpProject.csproj", GetResourceText("CSharpProject_CSharpProject_WithoutCSharpTargetsImported.csproj")));
+
+            var solution = MSBuildWorkspace.Create().OpenSolutionAsync(GetSolutionFileName(@"TestSolution.sln")).Result;
+            var project = solution.Projects.First();
+            var documents = project.Documents.ToList();
```

Comment:
This (`TestOpenProject_CSharp_WithoutCSharpTargetsImported_Succeeds()`) is pretty much modeled after existing tests suffixed `_Succeeds()` which do not contain an assertion but merely check that the calls do not throw. The other test (`TestOpenProject_CSharp_WithoutCSharpTargetsImported_DocumentsArePickedUp`) asserts that the documents are correctly populated from the project model.
I believe those are separate things to test but I can change it.